src/microbe_stage/Microbe.Contact.cs

```diff
@@ -505,10 +505,15 @@ public void Kill()
             }
         }
 
-        int chunksToSpawn = Math.Max(1, HexCount / Constants.CORPSE_CHUNK_DIVISOR);
+        // queues either 1 corpse chunk or a factor of the organelles
+        int chunksToSpawn = Math.Max(1, (organelles.Count + 1) / Constants.CORPSE_CHUNK_DIVISOR -
+            (1 / Constants.CORPSE_CHUNK_DIVISOR));
 
         var chunkScene = SpawnHelpers.LoadChunkScene();
```

Comment:
I changed the chunksToSpawn algorithm to use the number of organelles instead of hexes as fixing [this issue](https://github.com/Revolutionary-Games/Thrive/pull/3450#discussion_r907005849) resulted in the game crashing when a cell with large organelles dies, as it tries to spawn multiple chunks even though there's only one organelle

Reply:
Also trying to avoid divide by zero by factoring in/out a 1

Reply:
How? I don't understand...

Reply:
Here's a new idea I've gotten after sleeping on this: why don't we randomize the order when creating organellesAvailable list, and then create an enumerator from it which we just step once for each time a new chunk is needed? That avoids having to modify the list at all after creating it.

Reply:
Oops, my bad regarding div/0 - should be fine without factoring, will revise

Otherwise, while trying to spawn more chunks than organelles available it may continue with the loop and try to make a chunk without entering the foreach loop, throwing the generic exception `Chunk scene has not been loaded even though it should be loaded here` in Spawners.cs line 136

Reply:
> How? I don't understand...

div/0 is not possible, my bad. Should be resolved now.

> Here's a new idea I've gotten after sleeping on this: why don't we randomize the order when creating organellesAvailable list, and then create an enumerator from it which we just step once for each time a new chunk is needed? That avoids having to modify the list at all after creating it.

That is definitely a better solution.